### ISSUE TITLE
How do I install and run the project?

### ISSUE THREAD

**Issue by Tushn** [anAcc22/another_graph_editor#6]
Could you please provide instructions on how to set up and run this project?

**Comment by anAcc22** [comment]
Yup sure, I'll close this issue since it hasn't been active for a while. Maybe I'll add a short section in the README someday to show the local installation steps.
